### ISSUE TITLE
Cannot cast Power Infusion on a player actively on a Dragonriding Mount if they have the accessibility option enabled

### ISSUE THREAD

**Issue by seanpeters86** [SimCMinMax/WoW-BugTracker#1076]
**Game Version:**
10.0.2 live Build 46924

**Description:**
If a player enabled the Dragonriding ride-along feature they will be ineligible to get Power Infusion cast on them while mounted on a Dragonriding mount. This does not happen to players without the feature enabled.

**Logs, Images, Videos**
N/A

**Reproducible:**
yes

**Howto:**
1. Enable ride along
2. Notice you cannot get Power Infusion cast on you if mounted on a Dragonriding mount

**Comment by voidzone** [comment]
Affects players on any vehicle. ie Grand Expedition Yak
